src/client/components/PageHeaderComponent/PageHeader.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Search from '../SearchComponent/Search';
+import Button from '../Button/Button';
+import './PageHeader.styles.css';
+
+export default function PageHeader() {
+  return (
+    <div>
+      <div className="cv-loader">
+        <div>Uploaded Resume</div>        
+          <div className = "cv-button">
+            <Button buttonName="Upload New Cv" style = {{ backgroundColor: 'black', color: "white" }}/>
+          </div>
+        </div>
+      <div className="search-button">
+        <Search />
```

Comment:
Indentation. No need to fix it. BUT this should not be something i should need to comment at this stage! Use prettier that is what it is there for. Not indenting your code is a big red flag in my opinion. So remember that!